trunk/SUAVE/Input_Output/OpenVSP/vsp_write.py

```diff
@@ -673,6 +674,41 @@ def write_wing_conformal_fuel_tank(wing, wing_id,fuel_tank,fuel_tank_set_ind):
     # Offset
     vsp.SetParmVal(tank_id,'Offset','Design',offset)      
     
+    for key, fuselage in vehicle.fuselages.items():
```

Comment:
This section will need to be merged carefully since I believe it now conflicts with develop.

Reply:
Yeah, seriously.

Reply:
The merge is done